src/selectors/isValid.js

```diff
@@ -3,17 +3,25 @@ import createHasError from '../hasError'
 const createIsValid = structure => {
   const { getIn } = structure
   const hasError = createHasError(structure)
-  return (form, getFormState = state => getIn(state, 'form')) =>
+  return (form, getFormState, ignoreSubmitErrors) =>
```

Comment:
Hey, @erikras. I think this introduced a breaking change. Now, when calling `isValid('form')` like I did before (because `getFormState` was optional) it throws `TypeError: getFormState is not a function`